### PR TITLE
MAGECLOUD-2822: An error appears on SCD if it runs in parallel: Error while waiting for package deployed

### PR DESCRIPTION
**Pull request by oshmyheliuk**
### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-2822

### Manual testing scenarios

https://jira.corp.magento.com/browse/MAGECLOUD-101

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by NadiyaS — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA passed

**Comment by andriyShevtsov** [comment]
Validation was added. Retested again.  QA approved

File: dist/.magento.env.yaml [hunk 1]
```diff
@@ -58,6 +58,17 @@
 #            deploy:                                                                                                  #
 #              SCD_STRATEGY: "compact"                                                                                #
 #######################################################################################################################
+# SCD_MAX_EXECUTION_TIME - allows you to increase maximum expected execution time of static content deploy command.   #
```
Comment (meker12) [inline dist/.magento.env.yaml]:
```suggestion
# SCD_MAX_EXECUTION_TIME - allows you to increase the maximum expected execution time for the static content deploy command.   #
```

File: dist/.magento.env.yaml [hunk 2]
```diff
@@ -58,6 +58,17 @@
...
+#                          By default in Magento this value is set to 400 seconds but in some scenarios               #
```
Comment (meker12) [inline dist/.magento.env.yaml]: By default, Magento Commerce sets the maximum expected execution to 400 seconds, but in some scenarios you might need more time to complete the static content deployment for a Cloud project.

Comment (oshmyheliuk) [reply dist/.magento.env.yaml]: Fixed, thank you.

File: dist/.magento.env.yaml (hunk 1 above)
Comment (oshmyheliuk) [reply dist/.magento.env.yaml]: Fixed.

**Comment by shiftedreality** [comment]
Non-functional changes added. Approved.

**Pull request merged**